model/authorizations/v1/terms_review_request_type.model

```diff
@@ -23,4 +23,6 @@ struct TermsReviewRequest {
     SiteCode string
     // Defines the event code of the terms being checked
     EventCode string
+    // If false, only `terms_required` will be checked
+    check_optional_terms boolean
```

Comment:
Names of attributes should use camel case, so this should be "CheckOptionalTerms".